application/api/dos/tests/test_queries.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+
+from ..queries import can_user_edit_service
+
+
+class TestQueries(unittest.TestCase):
```

Comment:
Can we call the class TestCanUserEditService?  We can add more classes if we introduce more quieries. We can then shorten the names of the test methods....